.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "ros2_packages"]
 	path = ros2_packages
 	url = https://github.com/reazon-research/openarm_ros2.git
+[submodule "control/canfd_control/Vendor/STM32CubeH7"]
```

Comment:
Could you use "vendor" not "Vendor"?

It seems that auto generated files like Capital case but this is not auto generated path. Our non auto generated paths use snake_case. So "vendor" is better here.

Reply:
Thanks! I see!
Fixed!

https://github.com/reazon-research/OpenArm/compare/e630e00a3faac0e2cd8c4fa27e64e13351c62937..d1e2c29cfa8b67b7c291e787747a77b349f67e14